.prospector.yaml

```diff
@@ -0,0 +1,17 @@
+test-warnings: true
+doc-warnings: false
+member-warnings: false
+
+ignore-paths:
+  - docs
+ignore-patterns:
+  - (^|/)skip(this)?(/|$)
+autodetect: true
+#max-line-length: 88
+
+pylint:
+  disable:
+    # parameter_validations library methods give this error
+    - no-value-for-parameter
+  options:
+    max-args: 6
```

Comment:
comment ?

Reply:
default was 5 I think 6 will be more suitable. Do you mean why I made it 6? 5 is most of the times not enough. It is hard for me to make methods with less than 6 params is very difficult. In the future, we can evaluate by the way